### PR TITLE
[do-not-merge]

### PR DESCRIPTION
**Pull request by pieh**
## Description

just testing things out

### CODE REVIEW

File: e2e-tests/adapters/netlify.toml [hunk 1]
```diff
@@ -1,3 +1,9 @@
 [build]
   command = "npm run build"
-  publish = "public/"
\ No newline at end of file
+  publish = "public/"
+
+[build.environment]
+  NETLIFY_IMAGE_CDN = "true"
+
+[image]
```
Comment (kathmbeck) [inline e2e-tests/adapters/netlify.toml:8]: this should be `images`

File: e2e-tests/adapters/gatsby-node.ts [hunk 2]
```diff
@@ -62,3 +63,128 @@ export const createPages: GatsbyNode["createPages"] = ({
...
+  function createSchemaCustomization({ actions, schema, store }) {
+    actions.createTypes(`
+    type UnsplashImage implements Node & RemoteFile {
+      id: ID!
+    }
+
+    type DeployedLocalImage implements Node & RemoteFile {
+      id: ID!
+    }
+  `)
```
Comment (pieh) [inline e2e-tests/adapters/gatsby-node.ts:78]: this should be removed in favor of `MyRemoteFile` (which is copied from `e2e-tests/production`)

File: e2e-tests/adapters/gatsby-node.ts [hunk 3]
```diff
@@ -62,3 +63,128 @@ export const createPages: GatsbyNode["createPages"] = ({
...
+
```
Comment (pieh) [inline e2e-tests/adapters/gatsby-node.ts:79]: this should be removed in favor of MyRemoteFile (which is copied from e2e-tests/production)
```suggestion
```

File: e2e-tests/adapters/gatsby-node.ts [hunk 4]
```diff
@@ -62,3 +63,128 @@ export const createPages: GatsbyNode["createPages"] = ({
...
+        contentDigest: `2`,
+      },
+      url: imageURL,
+      filename: imageURL,
+      mimeType: `image/jpeg`,
+      width: 640,
+      height: 480,
+    })
+  }
+
```
Comment (pieh) [inline e2e-tests/adapters/gatsby-node.ts:148]: this should be removed in favor of MyRemoteFile (which is copied from e2e-tests/production)
```suggestion
```

File: e2e-tests/adapters/src/pages/routes/image-cdn.jsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      }
+    }
+    allDeployedLocalImage {
+      nodes {
+        id
+        gatsbyImage(width: 200, placeholder: NONE)
+      }
+    }
+  }
+`
```
Comment (pieh) [inline e2e-tests/adapters/src/pages/routes/image-cdn.jsx:53]: this should be removed in favor of `remote-file` (which is copied from e2e-tests/production)

**Comment by pieh** [comment]
Also `e2e-tests/adapters/static/images` dir should be removed

**Pull request closed without merging**